docs/reconcile-in-custom-namespace.md

```diff
@@ -0,0 +1,52 @@
+# Reconcile Cluster-API Objects In Custom Namespace <!-- omit in toc -->
+
+Cluster-api-provider-aws controllers by default, reconcile cluster-api objects
+across all namespaces in the cluster. However, it is possible to restrict
+reconciliation to a single namespace and this document tells you how.
+
+## Contents <!-- omit in toc -->
+
+- [Use cases](#use-cases)
+- [Configuring `cluster-api-provider-aws` controllers](#configuring-cluster-api-provider-aws-controllers)
+
+## Use cases
+
+- Grouping clusters into a namespace based on the AWS account will allow
+  managing clusters across multiple AWS accounts. This will require each
+  `cluster-api-provider-aws` controller to have credentials to their respective
+  AWS accounts. These credentials can be created as kubernetes secret and be
+  mounted in the pod at `/root/.aws`.
```

Comment:
```suggestion
  mounted in the pod at `/root/.aws` or as environment variables.
```

Reply:
Instead of attempting to list multiple ways of passing credentials, preferring to suggest one way of doing it.

Reply:
one way to do things is good for a tutorial but for documentation we should write down the features. If this is documented elsewhere I'm cool merging as is, otherwise I think it does make sense to let people know env var is a viable option

Reply:
I don't know where this is documented. I am OK including env vars as an option. I've committed @vincepri's suggestion.